staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -785,7 +785,7 @@ func installAPI(s *GenericAPIServer, c *Config) {
 	if c.EnableDiscovery {
 		s.Handler.GoRestfulContainer.Add(s.DiscoveryGroupManager.WebService())
 	}
-	if feature.DefaultFeatureGate.Enabled(features.APIPriorityAndFairness) {
+	if c.FlowControl != nil && feature.DefaultFeatureGate.Enabled(features.APIPriorityAndFairness) {
```

Comment:
so we do a nil check everywhere else within this file, but I think it was accidentally missed here; in actual execution, if the feature gate is enabled, `c.FlowControl` will never be nil. However, in this unit test, we don't set the feature gate, so `c.FlowControl` never gets set, so the following Install call fails. Thankfully a pretty harmless nil pointer dereference :sweat_smile:

Reply:
This and the fuzzer test were the only two that failed and I locally verified them. The other two CI failures (bazel and e2e-gce) seem to intermittent failures. Hopefully all green this time :D

Reply:
Great, thanks for the quick fix!